src/App/Form/Security/EditAvatarType.php

```diff
@@ -1,33 +1,35 @@
 <?php
 
-namespace App\Form;
+namespace App\Form\Security;
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
-use Symfony\Component\Security\Core\Validator\Constraints\UserPassword;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use App\Entity\User;
 
-class ProfileType extends AbstractType
+class EditAvatarType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('currentPassword', 'password', [
-                'mapped'      => false,
-                'constraints' => new UserPassword(),
+            ->add('file', FileType::class, [
+                'required' => false,
             ])
+            ->add('save', SubmitType::class)
         ;
     }
 
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
         $resolver->setDefaults([
-            'data_class' => 'App\Entity\User',
+            'data_class' => User::class,
         ]);
     }
 
     public function getName()
     {
-        return 'app_profile';
+        return 'edit_avatar';
```

Comment:
getName is not useful anymore, since we use FQCN instead of form names.

Reply:
What is FQCN ?

Reply:
Done on all forms